Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -2816,16 +2816,12 @@ instructionLabel(_i64_reinterpret_f64)
     nextIPIntInstruction()
 
 instructionLabel(_f32_reinterpret_i32)
-    pushInt32(t0)
-    fi2f t0, ft0
-    popFloat32(ft0)
```

Comment:
How do these end up as a `nop`? Shouldn't we be changing the format from 2's complement to IEEE-754?

Reply:
The deleted code obviously is bogus and a NOP as is, right? (push garbage -> pop garbage)

And its a nop on little-endian as far as I understand because this is supposed to reinterpret 32s bits as a IEEE float, which is what we do automatically when we `popFloat32`.

Reply:
There is also f32.convert_i32_s/u which would convert an int to a float which do `fi2f` AFAIU.